cmake/RAJAMacros.cmake

```diff
@@ -0,0 +1,85 @@
+macro(raja_add_executable)
+
+  set(options )
+  set(singleValueArgs NAME)
+  set(multiValueArgs SOURCES DEPENDS_ON)
+
+  cmake_parse_arguments(arg
+    "${options}" "${singleValueArgs}" "${multiValueArgs}" ${ARGN})
+
+  if (RAJA_ENABLE_CUDA) 
+    if (RAJA_ENABLE_CLANG_CUDA) 
+      add_executable(${arg_NAME} ${arg_SOURCES})
+      target_compile_options(${arg_NAME} PRIVATE 
+        -x cuda --cuda-gpu-arch=sm_35 --cuda-path=${CUDA_TOOLKIT_ROOT_DIR})
```

Comment:
It may be a good idea to have a CMake option for setting the compute capability (i.e., 'arch') so users can do this easily.